accredible_certificates.php

```diff
@@ -288,7 +288,8 @@ public static function hasCertificate($course_id, $user_id){
           $cert_exit = False;
           if(is_array($all_certificates)){
 			foreach ($all_certificates as $key => $cert) {
-			  if($cert->recipient->email == $user->user_email){
+			  $user_email = strtolower( $user->user_email );
+			  if($cert->recipient->email == $user_email){
```

Comment:
I think `hasCertificate` is not used anywhere anymore so alternatively, we could completely remove this method.